config/wiki_pages.yml

```diff
@@ -2346,7 +2346,7 @@ de:
     bridge: DE:Key:bridge
     bridge:movable: DE:Key:bridge:movable
     bridge:structure: DE:Key:bridge:structure
-    buiding: DE:Key:buiding
+    building: DE:Key:building
```

Comment:
?

Is this an api? (this would be wrong, since it'll collide w/ `building` below: